packages/babel-preset-css-prop/src/index.js

```diff
@@ -30,8 +30,8 @@ export default (
           sourceMap,
           autoLabel,
           labelFormat,
-          instances,
-          cssPropOptimization: true
+          cssPropOptimization: true,
```

Comment:
out of curiosity - why this even does exist? when one would not like to enable this optimization?

Reply:
Someone is using another library with a css prop. By default, we'll detect if `jsx` is imported and only do the transform then but that doesn't work with the auto-insertion so we have this option.